proxyclient/m1n1/proxy.py

```diff
@@ -669,78 +685,102 @@ def reboot(self):
         self.request(self.P_REBOOT, no_reply=True)
 
     def write64(self, addr, data):
+        '''Addr - write 8 bytes to given address'''
```

Comment:
Why not use the same syntax as the other commands, with the function name? i.e. `write64(addr, data) - ...`.

Reply:
Fixed in new update